x/ibc/23-commitment/types/merkle.go

```diff
@@ -104,6 +106,9 @@ func (p Path) String() string {
 // CONTRACT: provided path string MUST be a well formated path. See ICS24 for
 // reference.
 func ApplyPrefix(prefix exported.PrefixI, path string) Path {
+	if !validation.DefaultPathValidator(path) {
+		panic(fmt.Sprintf("Path: %s is malformed", path))
```

Comment:
Should I be returning an empty `Path` here instead?

Reply:
yes, + a wrapped error